feat: Add usage in readme (#2)
caicloud/ormb: README.md

<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*

- [ormb](#ormb)
  - [Installation](#installation)
  - [Usage](#usage)
    - [Save the model](#save-the-model)
    - [Push the model to a remote registry](#push-the-model-to-a-remote-registry)
    - [Pull the model from a remote registry](#pull-the-model-from-a-remote-registry)
    - [Export the model to the current directory](#export-the-model-to-the-current-directory)
  - [Community](#community)

<!-- END doctoc generated TOC please keep comment here to allow auto update -->

# ormb

> Manage Machine Learning/Deep Learning Models like Docker Images.

Reviewer: ```suggestion
Managing Machine Learning/Deep Learning Models like Docker Images.
```

size:    90.6 MiB
v1: saved
```

### Push the model to a remote registry

```bash
$ ormb push caicloud/resnetv2:v1
The push refers to repository [caicloud/resnetv2]
ref:     caicloud/resnetv2:v1
digest:  f51973c855608ab06d8f5e4333925a635f87f01ff992ffc5f9988f26d1da24e9
size:    90.6 MiB
v1: pushed to remote (1 layer, 90.6 MiB total)
```

### Pull the model from a remote registry

```bash
$ ormb pull caicloud/resnetv2:v1 
v1: Pulling from caicloud/resnetv2
ref:     caicloud/resnetv2:v1
digest:  f51973c855608ab06d8f5e4333925a635f87f01ff992ffc5f9988f26d1da24e9
size:    90.6 MiB
Status: Downloaded newer model for caicloud/resnetv2:v1
```

### Export the model to the current directory

```bash
$ ormb export caicloud/resnetv2:v1
ref:     localhost:5000/caicloud/resnetv2:v1
digest:  f51973c855608ab06d8f5e4333925a635f87f01ff992ffc5f9988f26d1da24e9
size:    90.6 MiB

$ tree ./resnet_v2_fp32_savedmodel_NCHW
resnet_v2_fp32_savedmodel_NCHW
├── 1538687196
│   ├── saved_model.pb
│   └── variables
│       ├── variables.data-00000-of-00001
│       └── variables.index
└── ormbfile.yaml

2 directories, 4 files
```

## Community

> ormb project is a part of Clever, which is a Cloud Native Machine Learning platform. We are going to open source the platform soon.

Reviewer: ```suggestion
ormb project is part of Clever, a Cloud Native Machine Learning platform. We are going to open source a community edition soon.
```
